docs/guides/development.md

```diff
@@ -1,82 +1,73 @@
 # Development guide
 
-This document covers development-related actions in ZKsync.
+This document outlines the steps for setting up and working with ZKsync.
 
-## Initializing the project
+## Installing the local ZK Stack CLI
 
-To setup the main toolkit, `zk`, simply run:
+To set up the local toolkit, begin by installing `zkstackup`. From the project's root directory, run the following
+commands:
 
-```
-zk
+```bash
+cd ./zkstack_cli/zkstackup
+./install --local
 ```
 
-You may also configure autocompletion for your shell via:
-
-```
-zk completion install
-```
+This installs `zkstackup` in your user binaries directory (e.g., `$HOME/.local/bin/`) and adds it to your `PATH`.
 
-Once all the dependencies were installed, project can be initialized:
+After installation, open a new terminal or reload your shell profile. From the project's root directory, you can now
+run:
 
-```
-zk init
+```bash
+zkstackup --local
 ```
 
-This command will do the following:
+This command installs `zkstack` from the current source directory.
 
-- Generate `$ZKSYNC_HOME/etc/env/target/dev.env` file with settings for the applications.
-- Initialize docker containers with `reth` Ethereum node for local development.
-- Download and unpack files for cryptographical backend.
-- Generate required smart contracts.
-- Compile all the smart contracts.
-- Deploy smart contracts to the local Ethereum network.
-- Create “genesis block” for server.
+> NOTE: Whenever you want to update you local installation with your changes, just rerun:
+>
+> ```bash
+> zkstackup --local
+> ```
+>
+> You might find convenient to add this alias to your shell profile:
+>
+> `alias zkstackup='zkstackup --path $ZKSYNC_HOME'`
```

Comment:
`ZKSYNC_HOME` doesn't exist anymore, let's not propose to use it

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/3134/commits/f98931a9f8cf6be00f55c6ba1096e4cef0ee4bee